document_sftp/models/document_sftp_root.py

```diff
@@ -59,6 +63,18 @@ def _open(self, path, flags, attr):
         raise NotImplementedError()
 
     @api.model
+    def _remove(self, path):
+        """Return file attributes"""
+        raise NotImplementedError()
+
+    @api.model
     def _lstat(self, path):
         """Return attributes about a link"""
         return self._stat(path)
+
+    @api.model
+    def _split_path(self, path):
+        """Return a list of normalized and stripped path components"""
+        # TODO: normalization
+        path = path.strip('/')
```

Comment:
no, the ssh protocol defines the path separator as bein a slash: https://tools.ietf.org/id/draft-ietf-secsh-filexfer-13.txt (section 6, file names)